osbuild-composer.spec

```diff
@@ -161,6 +161,7 @@ install -m 0755 -vp _bin/osbuild-composer-cloud-tests           %{buildroot}%{_l
 install -m 0755 -vp _bin/osbuild-auth-tests                     %{buildroot}%{_libexecdir}/tests/osbuild-composer/
 install -m 0755 -vp test/cmd/*                                  %{buildroot}%{_libexecdir}/tests/osbuild-composer/
 install -m 0755 -vp tools/image-info                            %{buildroot}%{_libexecdir}/osbuild-composer/
+install -m 0755 -vp tools/run-koji-container.sh                 %{buildroot}%{_libexecdir}/osbuild-composer/
```

Comment:
I wonder if we should install test helpers into a separate libexecdir, such as `osbuild-composer-tests` or `osbuild-composer/tests`

Reply:
Yeah, totally, that makes sense! (Maybe in a follow-up)